doc/classes/EditorInterface.xml

```diff
@@ -48,6 +48,14 @@
 				[b]Note:[/b] This returns the main editor control containing the whole editor, not the 2D or 3D viewports specifically.
 			</description>
 		</method>
+		<method name="get_editor_scale" qualifiers="const">
+			<return type="float">
+			</return>
+			<description>
+				Returns the actual scale of the editor UI. Can be used to adjust position and dimensions of the UI added by plugins.
```

Comment:
```suggestion
				Returns the actual scale of the editor UI ([code]1.0[/code] being 100% scale). This can be used to adjust position and dimensions of the UI added by plugins.
```